src/SelectField/SelectField.jsx

```diff
@@ -161,70 +168,172 @@ const SelectField = React.createClass({
     };
   },
 
+  componentDidMount() {
+    if (this.props.open) {
+      /* eslint-disable */
```

Comment:
I think it's needed, because we need to set an anchorElement for the popover.  If you know a better way, I'd be happy to drop in!